gammapy/cube/tests/test_core.py

```diff
@@ -133,8 +133,8 @@ def test_integral_flux_image(self):
         assert_allclose(new_image.data, image.data)
 
         # Test Header Keys
-        expected = [('CDELT1', 0.5), ('CDELT2', 0.5), ('NAXIS1', 61),
-                    ('NAXIS2', 21), ('CRVAL1', 0), ('CRVAL2', 0)]
```

Comment:
I mean that when I init `skymap.meta` as `self.wcs.sub(axes).to_header()` I don't have NAXIS fields so I thought they might be deprecated in skymap, removed them from test and wanted to verify